Tools/TestWebKitAPI/Tests/WebKitCocoa/SiteIsolation.mm

```diff
@@ -415,6 +415,22 @@ HTTPServer server({
     }
 }
 
+TEST(SiteIsolation, MultipleCrossSiteRedirects)
+{
+    HTTPServer server({
+        { "/example"_s, { "<iframe src='https://a.com/a'></iframe>"_s } },
+        { "/a"_s, { 302, { { "Location"_s, "https://b.com/b"_s } }, "redirecting..."_s } },
+        { "/b"_s, { 302, { { "Location"_s, "https://c.com/c"_s } }, "redirecting..."_s } },
+        { "/c"_s, { "arrived!"_s } }
+    }, HTTPServer::Protocol::HttpsProxy);
+
+    auto [webView, navigationDelegate] = siteIsolatedViewAndDelegate(server);
+    [webView loadRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:@"https://a.com/a"]]];
+    [navigationDelegate waitForDidFinishNavigation];
+    auto pool = webView.get().configuration.processPool;
+    EXPECT_EQ(pool._webProcessesCreated - pool._hasPrewarmedWebProcess, 2u);
+}
```

Comment:
EWS says this test fails on iOS. Should it be on macOS only?

`TestWebKitAPI.SiteIsolation.MultipleCrossSiteRedirects`